pyEpiabm/pyEpiabm/tests/test_unit/test_sweep/test_spatial_sweep.py

```diff
@@ -150,6 +150,23 @@ def test_find_infectees_fails(self, mock_dist, mock_nan, mock_logger):
         mock_logger.assert_called
         # test logger is called here
 
+    @mock.patch("logging.exception")
+    @mock.patch("numpy.nan_to_num")
+    @mock.patch("pyEpiabm.utility.DistanceFunctions.dist_euclid")
```

Comment:
Do we need these two patches?

Reply:
All mock patches removed none of them were actually needed